src/main/java/rs/hybridit/service/BookRentService.java

```diff
@@ -0,0 +1,12 @@
+package rs.hybridit.service;
+
+import java.util.Optional;
+import rs.hybridit.model.BookCopy;
+
+public interface BookRentService {
+
+	BookCopy rentBookCopy(Long bookId);
+
+	Optional<BookCopy> returnBookCopy(Long bookCopyId);
```

Comment:
return...?

Reply:
Why not? :D

Reply:
findById which already is JPA method? :P

Reply:
That method is for renturning book copy after renting it and it resets all fields (user, rentStart, rentEnd).